### PR TITLE
Nested groups test cases

### PR DESCRIPTION
**Pull request by marcocaberletti**
This PR introduce Selenium test case for IAM issue https://github.com/indigo-iam/iam/issues/88.

### CODE REVIEW

File: tests/group/nested_groups.robot [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  
+Create second level subgroup
+  Open Groups Add Group Dialog
+  Create group with parent  ${L1_GROUP}  ${PARENT_GROUP}
+  Open Groups Add Group Dialog
+  Create group with parent  ${L2_GROUP}  ${PARENT_GROUP}/${L1_GROUP}
+  ${fullname}=  Set Variable  ${PARENT_GROUP}/${L1_GROUP}/${L2_GROUP}
+  Find group in groups page  ${fullname}
+  Delete subgroup  ${fullname}
+
```
Comment (enricovianello) [inline tests/group/nested_groups.robot:38]: We could add the test "Delete not empty subgroup" and try to remove the first sublevel and check the error

File: tests/user/add_remove_credentials.robot [hunk 2]
```diff
@@ -15,6 +15,7 @@ ${TEST_OIDC_SUBJECT}  Test OIDC subject
```
Comment (enricovianello) [inline tests/user/add_remove_credentials.robot:15]: This is another fix that could be reported within the final merge/rebase commit description

**Review by enricovianello — APPROVED** [review]
